config/locales/ar.yml

```diff
@@ -65,6 +65,9 @@ ar:
     new:
       add_item: احذف العناصر
   spotlight:
+    bulk_actions:
+      delete_resources:
+        heading: حذف الموارد
```

Comment:
i filed a ticket: https://github.com/sul-dlss/dlme/issues/1313

Reply:
Thanks, @jmartin-sul. This shouldn't hold up merging. QNL will need to get involved and it will be slow. These are also fields that should only be visible to those with admin privileges and currently we have no need for these fields since we don;t have Arabic speaking curators.